### DISCUSSION TITLE
ERROR:asyncio:Task exception was never retrieved future: <Task finished name='Task-1034' coro=<WslinkHandler.sendWrappedMessage() done, defined at /root/.local/lib/python3.9/site-packages/wslink/backends/aiohttp/__init__.py:473> exception=ConnectionResetError('Cannot write to closing transport')>

### DISCUSSION THREAD

**Discussion by eagle201306** [Kitware/trame#106] [General]
ERROR:asyncio:Task exception was never retrieved
future: <Task finished name='Task-1034' coro=<WslinkHandler.sendWrappedMessage() done, defined at /root/.local/lib/python3.9/site-packages/wslink/backends/aiohttp/__init__.py:473> exception=ConnectionResetError('Cannot write to closing transport')>

When using -- host 0.0.0.0 configuration, the trame frequently refreshes the web page and reports the above error. How to solve this problem

**Comment by jourdain** [comment]
It really depend on your application and deployment setup and it has nothing to do with using `0.0.0.0` address. Most likely your forwarding ws proxy that is miss configured.
